app/models/name/taxonomy.rb

```diff
@@ -468,9 +468,9 @@ def best_matches_from_array(names, includes = [])
     # Now allows includes, for batch lookup of Naming email interested parties
     # GOTCHA: `search_name` cannot be used as a field in this AR where clause
     def batch_lookup_all_matches(name_or_names, includes = [])
-      Name.includes(includes).where(Name[:search_name].in(name_or_names)).
-        or(Name.where(Name[:text_name].in(name_or_names))).
-        with_correct_spelling
+      Name.unscoped.where(Name[:search_name].in(name_or_names)).
+        or(Name.unscoped.where(Name[:text_name].in(name_or_names))).
+        with_correct_spelling.includes(includes)
```

Comment:
I believe this is an appropriate use of `.unscoped`, as is the one below. 

These are one-off queries that should not be otherwise scoped and do not rely on order.